src/db/postgres/lrsql/postgres/sql/ddl.sql

```diff
@@ -280,3 +280,40 @@ CREATE TYPE scope_enum AS ENUM (
   'profile',
   'profile/read');
 ALTER TABLE IF EXISTS credential_to_scope ALTER COLUMN scope TYPE scope_enum USING (scope::scope_enum);
+
+
+/* Migration 2022-05-08-00 - Add timestamp to xapi_statement */
+
+-- :name query-xapi-statement-timestamp-exists
+-- :command :query
+-- :result :one
+-- :doc Query to see if `xapi_statement.timestamp` exists.
+SELECT 1 FROM information_schema.columns WHERE table_name = 'xapi_statement' AND column_name = 'timestamp';
+
+-- :name alter-xapi-statement-add-timestamp!
+-- :command :execute
+-- :doc Add `xapi_statement.timestamp` to allow easier timestamp access.
+ALTER TABLE xapi_statement ADD COLUMN timestamp TIMESTAMPTZ
```

Comment:
So statement `stored` and `timestamp` are `TIMESTAMPZ`, but `last_modified` in other tables are just plain `TIMESTAMP`. That can get confusing.

Reply:
Well as ive explained, you need TZ to actually represent normalized zulu

Reply:
Discussed on Slack - such a concern would need to be addressed in a future PR.